include/SugarFields/Parsers/Rules/ActivitiesParseRule.php

```diff
@@ -46,25 +46,6 @@
 
 class ActivitiesParseRule extends BaseRule
 {
-    public function __construct()
```

Comment:
Not big deal by removing this constructor as it does nothing, but does not follow the standard of how has been done on other classes.